packages/common/src/models/USDCTransactions.ts

```diff
@@ -38,46 +27,12 @@ export type USDCPurchaseDetails = {
   createdAt: string
 }
 
-export type USDCPurchaseContentMetadata = {
-  discriminator: USDCTransactionType.PURCHASE_CONTENT
-  amount: StringUSDC
-  senderUserId: string
-  receiverUserId: string
-  purchaseType: USDCContentPurchaseType
-  contentId: number
-}
-
-export type USDCTransferMetadata = {
-  discriminator: USDCTransactionType.TRANSFER
-  amount: StringUSDC
-  destination: string
+export type USDCTransactionDetails = {
+  signature: string
+  transactionType: USDCTransactionType
+  method: USDCTransactionMethod
+  transactionDate: string
+  change: StringUSDC
+  balance: StringUSDC
+  metadata?: Nullable<object> | string
 }
-
```

Comment:
These types were originally added when the tech spec document was done, but they weren't in use yet. The actual types coming back from the API are a little different. And we don't really need the discriminated union.